README.md

```diff
@@ -432,8 +432,8 @@ quot15 = '''Here are fifteen quotation marks: """""""""""""""'''
 # apos15 = '''Here are fifteen apostrophes: ''''''''''''''''''  # INVALID
 apos15 = "Here are fifteen apostrophes: '''''''''''''''"
 
-# 'That's still pointless', she said.
-str = ''''That's still pointless', she said.'''
+# 'That,' she said, 'is still pointless.'
+str = ''''That,' she said, 'is still pointless.''''
```

Comment:
This is for #725?

Reply:
Oh, yes, it's 725, 727 is this PR...

Reply:
I have updated the hash number in PR explain. (Translating 1.0.0 and updating my parser has worn me out and I'm a bit dazed now... But still so excited to see 1.0.0 come)